# Request
uint8 id
---
# Response
int16 current
